### DISCUSSION TITLE
Dolby Vision to HDR10+ possible?

### DISCUSSION THREAD

**Discussion by wyup** [R3S3t9999/DoVi_Scripts#12] [Ideas]
Hello,
dovi_tool developer quietvoid says DV to HDR10+ transcoding is possible by decoding DV scenes based on DV metadata and generate histograms and HDR10+ compatible metadata by means of ffmpeg libplacebo library and VapourSynth/AviSynth+

[https://github.com/quietvoid/hdr10plus_tool/discussions/67](https://github.com/R3S3t9999/DoVi_Scripts/discussions/67)

Would you ever tackle this? Thanks for all development and shared tools/guides.

**Comment by R3S3t9999** [comment]
I can take a look but since the average pq is calculated quite differently in HDR10plus, you would get much better metadata if y[ou generate HDR10plus in Resolve](https://www.youtube.com/watch?v=ZdFiqATo_j8) IMO
